editor/scene/canvas_item_editor_plugin.cpp

```diff
@@ -3797,14 +3797,14 @@ void CanvasItemEditor::_draw_selection() {
 				xform.xform(rect.position + Vector2(0, rect.size.y))
 			};
 
-			Color c = Color(1, 0.6, 0.4, 0.7);
+			Color selection_square_color = EDITOR_GET("editors/2d/selection_square_color");
```

Comment:
The color could be cached, to not fetch the setting every time.
Also
```suggestion
			Color selection_square_color = item_locked ? EDITOR_GET("editors/2d/locked_selection_square_color") : EDITOR_GET("editors/2d/selection_square_color");
```